packages/features/form-builder/Components.tsx

```diff
@@ -205,7 +205,16 @@ export const Components: Record<FieldType, Component> = {
       if (!props) {
         return <div />;
       }
-      return <Widgets.TextWidget type="email" id={props.name} noLabel={true} {...props} />;
```

Comment:
We use input field in the rest of this components file - not sure why we used widget here.